Pop-Grupa-A-Frontend/src/components/navigationBar/NavBars.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Navbar from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
+import 'bootstrap/dist/css/bootstrap.css';
+
+
+function LoginRegistrationNavBar() {
```

Comment:
Myślę, że lepiej by było stosować arrow function tutaj, żeby to było zgodne z pozostałym kodem @strachob